.github/iwyu.imp

```diff
@@ -16,8 +16,8 @@
   { include: ['<Acts/Geometry/detail/DefaultDetectorElementBase.hpp>', private, '<Acts/Geometry/DetectorElementBase.hpp>', public] },
 
   # Eigen
-  { include: ['@<Eigen/(src/)?(.*?)/.*>', private, '<Eigen/$2>', public] },
-  { include: ['@<Eigen/src/plugins/.*>', private, '<Eigen/Core>', public] },
+  { include: ['@[<"]Eigen/(src/)?(.*?)/.*[>"]', private, '<Eigen/$2>', public] },
+  { include: ['@[<"]Eigen/src/plugins/.*[>"]', private, '<Eigen/Core>', public] },
```

Comment:
@wdconinc FYI this works

Reply:
Nice, even more simplification of that file.